### PR TITLE
Page titles

### PR DESCRIPTION
**Pull request by jkmaswell**
## Summary

Adding title component to project

## Screenshots

image

## Trello Card

https://trello.com/c/ABheqvdT/21-page-titles

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,7 +10,10 @@
 </head>
 
 <body>
-  <h1 class="title">Hello World</h1>
+  <h1 class="title">
+    <i class="title-icon"></i>
```
Comment (FrankIglesias) [inline index.html]: use a pseudo element

File: index.html [hunk 2]
```diff
@@ -10,7 +10,10 @@
...
+    <span class="title-content">Front-End Tools</span>
```
Comment (FrankIglesias) [inline index.html]: delete this and just leave the text

File: scss/fonts.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,700');
+
+$font-family: 'Roboto', sans-serif;
```
Comment (FrankIglesias) [inline scss/fonts.scss]: define the font family to all the content here

Comment (FrankIglesias) [reply scss/fonts.scss]: add empty line

File: scss/fonts.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,700');
```
Comment (FrankIglesias) [inline scss/fonts.scss]: @import "https://fonts.googleapis.com/css?family=Roboto:300,400,700"

File: scss/mixins.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+@mixin font($font) {
+  font-family: $font;
+}
```
Comment (FrankIglesias) [inline scss/mixins.scss]: add empty line and define for all text this prop in fonts.scss

File: index.html [hunk 6]
```diff
@@ -10,7 +10,7 @@
 </head>
 
 <body>
-  <h1 class="title">Hello World</h1>
+  <h1 class="title">Front-End Tools</h1>
```
Comment (FrankIglesias) [inline index.html]: delete space at the bottom

File: scss/mixins.scss (hunk 5 above)
Comment (jkmaswell) [reply scss/mixins.scss]: Done

Comment (FrankIglesias) [reply scss/mixins.scss]: :arrow_up:

File: scss/titles.scss [hunk 7]
```diff
@@ -0,0 +1,14 @@
+.title {
+  align-items: center;
+  display: flex;
+  color: $black;
+  font-size: 1.45rem;
```
Comment (FrankIglesias) [inline scss/titles.scss]: use px

File: scss/titles.scss [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+
+  &::before {
+    background: url("assets/triangle.svg") center center no-repeat;
+    content: '';
+    height: 90px;
+    margin-right: -1.1rem;
```
Comment (FrankIglesias) [inline scss/titles.scss]: same

File: index.html (hunk 6 above)
Comment (jkmaswell) [reply index.html]: Done

**Pull request merged**